src/sentry/models/organizationmember.py

```diff
@@ -223,10 +223,13 @@ class OrganizationMember(Model):
     # Deprecated -- no longer used
     type = BoundedPositiveIntegerField(default=50, blank=True)
 
+    # These attributes are replicated via USER_UPDATE category outboxes for the user object associated with the user_id
+    # when it exists.
     user_is_active = models.BooleanField(
         null=False,
         default=True,
     )
+    user_email = models.CharField(max_length=75, null=True, blank=True)
```

Comment:
how is this different from the `email` above? Why not just use that?

Reply:
Leave some commentary here explaining the semantic differences.

Reply:
Added comments.

Reply:
If a user has multiple emails, is this the primary email for the user? Do we have outbox messages for user email change to keep this in sync?